CharlieBackend.Api.UnitTes/TEst.cs

```diff
@@ -0,0 +1,24 @@
+﻿using AutoMapper;
+using CharlieBackend.Core.DTO.Lesson;
+using CharlieBackend.Core.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace CharlieBackend.Api.UnitTest
+{
+    public class TEst
+    {
+        [Fact]
+        public void CheckDto()
+        { 
+        var configuration = new MapperConfiguration(cfg =>
+  cfg.CreateMap<Lesson, CreateLessonDto>()
+                .ForMember(destination => destination.LessonVisits, conf => conf.MapFrom(x => x.Visits.Select(x => x.Lesson.Visits))));
```

Comment:
too much symbols in the line

Reply:
TEst.cs is debug temporary file, which is absent in this pull request last commit.